### PR TITLE
[NEW] List the room members.

### LINKED ISSUE #62: [NEW] REST API to list members of a channel/group/dm

### PR DESCRIPTION
**Pull request by filipedelimabrito**
Closes #62 

Note: Need to add the JSON return from the server when getting the member list (to be used on the tests).

### CODE REVIEW

**Comment by filipedelimabrito** [comment]
@rafaelks To makes the thing speed up, do you know the JSON response from the server when calling this API?

See this [example result](https://rocket.chat/docs/developer-guides/rest-api/chat/delete/) for example.

We need to add this to our doc, as pointed [here](https://rocket.chat/docs/contributing/documentation/missing-and-outdated-list/index.html#channelsmembers)

**Comment by rafaelks** [comment]
@filipedelimabrito I guess it's pretty fast for you to get that by running your SDK method and logging out the result? Or do you need something different? 🤔

File: core/src/main/kotlin/chat/rocket/core/internal/model/ChatRoomPayload.kt [hunk 1]
```diff
@@ -1,6 +1,7 @@
 package chat.rocket.core.internal.model
 
+import com.squareup.moshi.Json
 import se.ansman.kotshi.JsonSerializable
 
 @JsonSerializable
-class ChatRoomPayload(val rid: String)
\ No newline at end of file
+class ChatRoomPayload(@Json(name = "rid") val roomId: String)
```
Comment (luciofm) [inline core/src/main/kotlin/chat/rocket/core/internal/model/ChatRoomPayload.kt]: Use data class, you get `toString()` `equals()`, `hashCode()` and component functions for free.

Comment (filipedelimabrito) [reply core/src/main/kotlin/chat/rocket/core/internal/model/ChatRoomPayload.kt]: Great! I just forget this detail! Data class!! 
Thanks fm!

**Pull request merged**